EchoPro/core.py

```diff
@@ -0,0 +1,174 @@
+        "dataprep.y_offset": np.float64 ,
+        "vario.range": np.float64 ,
+        "vario.res": np.float64 ,
+        "vario.vario": np.float64 ,
+        "vario.corr": np.float64 ,
+        "vario.nugt": np.float64 ,
+        "vario.sill": np.float64 ,
+        "vario.lscl": np.float64 ,
+        "vario.powr": np.float64 ,
+        "vario.hole": np.float64 ,
+        "vario.model": np.float64 ,
+        "vario.ytox_ratio": np.float64 ,
+        "vario.ztox_ratio": np.float64 ,
+        "vario.dim": np.float64 ,
+        "krig.x_res": np.float64 ,
+        "krig.y_res": np.float64 ,
+        "krig.proc_opt": np.float64 ,
+        "krig.load_para": np.float64 ,
+        "krig.vario_para": np.float64 ,
+        "krig.krig_para": np.float64 ,
+        "krig.both_para": np.float64 ,
+        "krig.load_griddata_file": np.float64 ,
+        "krig.xmin0": np.float64 ,
+        "krig.xmax0": np.float64 ,
+        "krig.dx0": np.float64 ,
+        "krig.ymin0": np.float64 ,
+        "krig.ymax0": np.float64 ,
+        "krig.dy0": np.float64 ,
+        "krig.nx": np.float64 ,
+        "krig.ny": np.float64 ,
+        "krig.nz": np.float64 ,
+        "krig.dx": np.float64 ,
+        "krig.dy": np.float64 ,
+        "krig.dz": np.float64 ,
+        "krig.model": np.float64 ,
+        "krig.scheme": np.float64 ,
+        "krig.blk_nx": np.float64 ,
+        "krig.blk_ny": np.float64 ,
+        "krig.blk_nz": np.float64 ,
+        "krig.srad": np.float64 ,
+        "krig.kmin": np.float64 ,
+        "krig.kmax": np.float64 ,
+        "krig.elim": np.float64 ,
+        "krig.eps": np.float64 ,
+        "krig.ratio": np.float64 ,
+        "krig.xmin": np.float64 ,
+        "krig.xmax": np.float64 ,
+        "krig.ymin": np.float64 ,
+        "krig.ymax": np.float64 ,
+    } ,
+}
+
+LAYER_NAME_MAP = {
+    """
+    ``LAYER_NAME_MAP`` is a hard-coded dictionary that aids in re-mapping the data tree
+    structure of various attributes contained within a generated ``Survey`` class object. 
+    Each dictionary comprises the same dataset names contained within the configuration file.
+    Nested within each of these dictionaries include three variables: 
+        name (string): the name of the attribute appended to a ``Survey`` class object.
+        data (list[string]): the name of the expected data layer, which is formatted as a dataframe.
+        superlayer (list[string]): layer names that are prepended onto the nested dictionary structure of the data attribute.
+    While this reference library is only required in the sense that it is hard-coded into the
+    the current implementation concerning loading data into a ``Survey`` class object, it may be 
+    helpful for organizing data with more parsimonious and representative names. It may also be
+    helpful for future extensions enabling users to set additional data attributes that aren't 
+    included in the ``Survey`` class initialization and parameterization (when the object is generated).
```

Comment:
Just trying to get it into a format that is similar to numpydoc, though these won't really be rendered.
```suggestion
    ``LAYER_NAME_MAP`` is a hard-coded dictionary that aids in re-mapping the data tree
    structure of various attributes contained within a generated ``Survey`` class object. 
    Each dictionary comprises the same dataset names contained within the configuration file.
    
    Nested within each of these dictionaries include three variables: 
        name : string)
            The name of the attribute appended to a ``Survey`` class object.
        data : list[string]
            The name of the expected data layer, which is formatted as a dataframe.
        superlayer : list[string]
            Layer names that are prepended onto the nested dictionary structure of the data attribute.

    While this reference library is only required in the sense that it is hard-coded into the
    the current implementation concerning loading data into a ``Survey`` class object, it may be 
    helpful for organizing data with more parsimonious and representative names. It may also be
    helpful for future extensions enabling users to set additional data attributes that aren't 
    included in the ``Survey`` class initialization and parameterization (when the object is generated).
```